File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/UpdateExternalidRequest.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class UpdateExternalidRequest {
    @JsonProperty("ExternalId")
    public String externalId;
    @JsonProperty("PackageLabel")
    public String packageLabel;
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/UpdateExternalidRequest.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class UpdateExternalidRequest(
    @JsonProperty("ExternalId")
    val externalId: String? = null,
    @JsonProperty("PackageLabel")
    val packageLabel: String? = null
)
